### DISCUSSION TITLE
CSLA 6 prerelease online

### DISCUSSION THREAD

**Discussion by rockfordlhotka** [MarimerLLC/csla#2624] [Announcements]
I'm excited to let you all know that the first prerelease of CSLA 6 is now online in NuGet!

A lot of people have put a lot of work into getting us to this point, and as we near final release I'll be assembling a list of everyone for serious kudos!

Right now, here's a blog post with info about the biggest changes that impact anyone looking at CSLA 6.

https://blog.lhotka.net/2021/11/29/CSLA-6-First-Prerelease

**Comment by vijaygill** [comment]
Thanks to all the developers for their wonderful work!
@rockfordlhotka - Will there be a new edition of your ebook on CSLA and Blazor? I will be glad to buy and add that to my collection :)

I am asking so because I started making changes to my current project but I am facing a few issues.

**Reply by rockfordlhotka** [reply]
Hi @vijaygill, yes, I do plan to update the books.

This is the first prerelease, so if you are encountering issues they might be bugs. Please let us know about the issues so we can help resolve them.

**Reply by vijaygill** [reply]
> Hi @vijaygill, yes, I do plan to update the books.
> 
> This is the first prerelease, so if you are encountering issues they might be bugs. Please let us know about the issues so we can help resolve them.

Thanks @rockfordlhotka!!! Sorry I did not mean issues in your work.  The code needed quite a bit of changes especially around start-up of the applications. I have a test C# Solution which has DataPortalServer, Blazor (Server side), Client, Blazor WebAssembly client along with other projects (like DAL, Shared UI). The implementation is based on your CSLA WebAssembly ebook.

After applyong some elbow-grease, the solution is back in running state but I would like official guidelines. But from now-on there will be more testing and I will report if I encounter any issues.

Thanks for your reply. I am looking forward to next edition of your ebook.

Vijay

**Reply by rockfordlhotka** [reply]
The BlazorExample sample is your best guide right now.

**Reply by vijaygill** [reply]
> The BlazorExample sample is your best guide right now.

Thank you Sir! And that's what I followed to get things going.
